### PR TITLE
EDNAMODE TOKEN

### PR DESCRIPTION
**Pull request by Gautambnsl**
# Hello MELD Team! &#128736;
I am submitting this pull request to add details for the EDNAMODE MEME (EDNAMODE) in the repository. I have followed the prescribed guidelines and ensured that all necessary details are accurate and align with the required data structure. Below is an overview of the token and the associated changes.

- [] I have checked that the format of my PR follows the contribution guidelines

### Token Overview &#128221;
- Token Name: EDNAMODE MEME 
- Token Ticker: EDNAMODE
- Blockchain: MELD
- Token Type: ERC20
- Primary Use Case: Meme coin

### Organization Details &#127970;
- Organization Name: Ednamode
- Website: https://ednamodememe.com/
- Mission & Vision: A meme with a utility where users can play to burn EDNA tokens, Stake EDNA and receive exclusive EDNA NFT's

### CODE REVIEW

File: networks/meld/tokens/0x660296b5fb3fd49319e7d7655e4ad0cea33ffbc7.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "orgName": "Ednamode",
+    "orgUrl": "https://ednamodememe.com/",
+    "tokenAddress": "0x660296b5fb3fd49319e7d7655e4ad0cea33ffbc7",
+    "tokenColorTheme": 3,
+    "tokenIcon": "/static/tokens/EDNAMODE.svg",
+    "tokenName": "EDNAMODE MEME",
+    "tokenTicker": "EDNAMODE",
+    "tokenType": "ERC20",
+    "tags": [
+      "EDNAMODE"
```
Comment (thuyvu17) [inline networks/meld/tokens/0x660296b5fb3fd49319e7d7655e4ad0cea33ffbc7.json:14]: This tag isn't supported at the moment. Just leave it blank for now.

Comment (Gautambnsl) [reply networks/meld/tokens/0x660296b5fb3fd49319e7d7655e4ad0cea33ffbc7.json]: Done

File: networks/meld/tokens/0x660296b5fb3fd49319e7d7655e4ad0cea33ffbc7.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "orgUrl": "https://ednamodememe.com/",
+    "tokenAddress": "0x660296b5fb3fd49319e7d7655e4ad0cea33ffbc7",
+    "tokenColorTheme": 3,
+    "tokenIcon": "/static/tokens/EDNAMODE.svg",
+    "tokenName": "EDNAMODE MEME",
+    "tokenTicker": "EDNAMODE",
+    "tokenType": "ERC20",
+    "tags": [
+     
+    ]
```
Comment (thuyvu17) [inline networks/meld/tokens/0x660296b5fb3fd49319e7d7655e4ad0cea33ffbc7.json:15]:
```suggestion
    "tags": []
```

Comment (Gautambnsl) [reply networks/meld/tokens/0x660296b5fb3fd49319e7d7655e4ad0cea33ffbc7.json]: Done

**Review by narrasumanth — APPROVED** [review]
Looks good

**Review by thuyvu17 — APPROVED** [review]

**Pull request merged**